artiq/coredevice/ad53xx.py

```diff
@@ -108,8 +114,8 @@ def off(self):
 
 
 class AD53xx:
-    """Analog devices AD53[67][0123] family of multi-channel Digital to Analog
-    Converters.
+    """Analog devices AD53[67][0123] family of 16-bit multi-channel Digital to
```

Comment:
There are several 14 bit parts in that family which are also supported by this driver. The words are still 16 bit aligned and word-sized but they are not 16-bit resolution. This comment refers to the resolution.

Reply:
I noticed this. However the 14-bit devices are not supported as voltage_to_mu is only valid for the 16-bit resolution family members.

for 16-bit members:
![image](https://user-images.githubusercontent.com/14295481/81410107-200adb80-9138-11ea-879f-9048465880f6.png)
for 14-bit members:
![image](https://user-images.githubusercontent.com/14295481/81410205-403a9a80-9138-11ea-8ae4-4cbe35f52a4a.png)

Reply:
I now see, the code is correted in the SPI transfer.
![image](https://user-images.githubusercontent.com/14295481/81413125-d5d82900-913c-11ea-9424-73cd913875d4.png)